### PR TITLE
[FIX] fixed tips for docker compose & translations

### PR DESCRIPTION
**Pull request by lavi02**
## Context 

* Adjust the order of docker compose commands when backing up data with docker compose.
  * If follow the previous commands leads to initialization issues on Ubuntu 22.04 LTS version (AWS EC2).
* Added missing Korean README translation

## Choices

* need to stop first before changing something.

## Checklist before requesting a review

* [v] I have performed a self-review of my code
* [v ] I've added documentation about this change to the README.
* [v] I've not introduced breaking changes.

### CODE REVIEW

File: docs/kr/README.md [hunk 1]
```diff
@@ -60,14 +61,33 @@ services:
...
+      - TZ=KST
       - ADMIN_PASSWORD="adminPasswordHere"
       - COMMUNITY=false # 커뮤니티 서버 탐색기에 서버가 표시 되는 것을 허용합니다 (USE WITH SERVER_PASSWORD 와 함께 사용하는 것을 권장합니다)
       - SERVER_NAME="World of Pals"
     volumes:
       - ./palworld:/palworld/
 ```
 
+또 다른 방법으로, .env.example 파일을 .env라는 새 파일로 복사할 수 있습니다.
+필요에 따라 수정하고, 올바른 값을 확인하려면 환경 변수 섹션을 확인하세요. 아래와 같이 docker-compose.yml을 수정하세요:
```
Comment (DaeHyeoNi) [inline docs/kr/README.md:73]: need hyperlink : `.env.exmaple`, `환경 변수`, `docker-compose.yml`
need bold: `.env`

File: docs/kr/README.md [hunk 2]
```diff
@@ -213,6 +245,94 @@ docker exec -it palworld-server restore
 >
 > [사용하기](#사용하기)에서 제공된 Docker 실행 명령어와 Docker Compose 파일 예시는 이미 필요한 정책을 적용하고 있습니다.
 
+## 수동으로 백업에서 복원하기
+
+/palworld/backups/에서 복원하고자 하는 백업을 찾아서 압축을 풉니다.
```
Comment (DaeHyeoNi) [inline docs/kr/README.md:250]: need backtick in path
```suggestion
`/palworld/backups/` 에서 복원하고자 하는 백업을 찾아서 압축을 풉니다.
```

File: docs/kr/README.md [hunk 3]
```diff
@@ -213,6 +245,94 @@ docker exec -it palworld-server restore
...
+
+```bash
+docker compose down
+```
+
+`palworld/Pal/Saved/SaveGames/0/<old_hash_value>`에 위치한 기존 저장 데이터 폴더를 삭제합니다.
+
+새롭게 압축 해제된 저장 데이터 폴더 `Saved/SaveGames/0/<new_hash_value>`의 내용을 `palworld/Pal/Saved/SaveGames/0/<new_hash_value>`로 복사합니다.
+
+`palworld/Pal/Saved/Config/LinuxServer/GameUserSettings.ini`` 안의 DedicatedServerName을 새 폴더 이름으로 교체합니다.
```
Comment (DaeHyeoNi) [inline docs/kr/README.md:261]:
```suggestion
`palworld/Pal/Saved/Config/LinuxServer/GameUserSettings.ini` 안의 DedicatedServerName을 새 폴더 이름으로 교체합니다.
```

File: docs/kr/README.md [hunk 4]
```diff
@@ -213,6 +245,94 @@ docker exec -it palworld-server restore
...
+
+## Cron을 이용한 자동 백업 설정
+
+서버는 TZ로 설정된 시간대에 따라 매일 자정에 자동으로 백업됩니다.
+
+BACKUP_ENABLED를 설정하여 자동 백업을 활성화하거나 비활성화합니다 (기본값은 활성화됨).
+
+BACKUP_CRON_EXPRESSION은 cron 표현식으로, Cron 표현식에서는 작업을 실행할 간격을 정의합니다.
+
+> [!팁]
```
Comment (DaeHyeoNi) [inline docs/kr/README.md:281]:
```suggestion
> [!TIP]
```

File: docs/kr/README.md [hunk 5]
```diff
@@ -213,6 +245,94 @@ docker exec -it palworld-server restore
...
+예시: `0 2 * * *`로 BACKUP_CRON_EXPRESSION을 설정하면, 백업 스크립트는 매일 새벽 2시에 실행됩니다.
+
+## Cron을 이용한 자동 업데이트 설정
+
+이 서버에서 자동 업데이트를 사용하려면 다음 환경 변수들을 `true`로 **설정해야 합니다**:
+
+* RCON_ENABLED
+* UPDATE_ON_BOOT
+
+> [!중요]
```
Comment (DaeHyeoNi) [inline docs/kr/README.md:296]:
```suggestion
> [!IMPORTANT]
```

File: docs/kr/README.md [hunk 6]
```diff
@@ -213,6 +245,94 @@ docker exec -it palworld-server restore
...
+> [supercronic](https://github.com/aptible/supercronic#crontab-format) 또는
+> [Crontab Generator](https://crontab-generator.org)를 참조하세요.
+
+AUTO_UPDATE_CRON_EXPRESSION을 설정하여 기본 스케줄을 변경합니다.
+
+## Cron을 이용한 자동 재부팅 설정
+
+이 서버에서 자동 재부팅을 사용하려면 RCON_ENABLED를 활성화해야 합니다.
+
+> [!중요]
```
Comment (DaeHyeoNi) [inline docs/kr/README.md:318]:
```suggestion
> [!IMPORTANT]
```

File: docs/kr/README.md [hunk 7]
```diff
@@ -213,6 +245,94 @@ docker exec -it palworld-server restore
...
+> docker 재시작 정책이 always 또는 unless-stopped로 설정되어 있지 않다면, 서버는 종료되고
+> 수동으로 다시 시작해야 합니다.
+>
+> [How to Use](#how-to-use)에서 이미 필요한 정책을 사용하는 예시 docker run 명령어와 docker compose 파일이 있습니다.
+
+AUTO_REBOOT_ENABLED를 설정하여 자동 재부팅을 활성화하거나 비활성화합니다 (기본값은 비활성화됨).
+
+AUTO_REBOOT_CRON_EXPRESSION은 cron 표현식으로, Cron 표현식에서는 작업을 실행할 간격을 정의합니다.
+
+> [!팁]
```
Comment (DaeHyeoNi) [inline docs/kr/README.md:329]:
```suggestion
> [!TIP]
```

File: docs/kr/README.md [hunk 8]
```diff
@@ -213,6 +245,94 @@ docker exec -it palworld-server restore
...
+> docker 재시작 정책이 `always` 또는 `unless-stopped`로 설정되어 있지 않다면, 서버는 종료되고
+> 수동으로 다시 시작해야 합니다.
+>
+> [How to Use](#how-to-use)에서 이미 필요한 정책을 사용하는 예시 docker run 명령어와 docker compose 파일이 있습니다.
+
+AUTO_UPDATE_ENABLED를 설정하여 자동 업데이트를 활성화하거나 비활성화합니다 (기본값은 비활성화됨).
+
+AUTO_UPDATE_CRON_EXPRESSION은 cron 표현식으로, Cron 표현식에서는 작업을 실행할 간격을 정의합니다.
+
+> [!팁]
```
Comment (DaeHyeoNi) [inline docs/kr/README.md:307]:
```suggestion
> [!TIP]
```

**Review by DaeHyeoNi — CHANGES_REQUESTED** [review]

File: docs/kr/README.md [hunk 9]
```diff
@@ -60,14 +61,33 @@ services:
...
-      - TZ=UTC
+      - TZ=KST
       - ADMIN_PASSWORD="adminPasswordHere"
       - COMMUNITY=false # 커뮤니티 서버 탐색기에 서버가 표시 되는 것을 허용합니다 (USE WITH SERVER_PASSWORD 와 함께 사용하는 것을 권장합니다)
       - SERVER_NAME="World of Pals"
     volumes:
       - ./palworld:/palworld/
 ```
 
+또 다른 방법으로, **.env.example** 파일을 **.env**라는 새 파일로 복사할 수 있습니다.
```
Comment (DaeHyeoNi) [inline docs/kr/README.md:72]:
```suggestion
또 다른 방법으로, [.env.example](/.env.example) 파일을 [.env](/.env)라는 새 파일로 복사할 수 있습니다.
```

File: docs/kr/README.md [hunk 10]
```diff
@@ -60,14 +61,33 @@ services:
...
+필요에 따라 수정하고, 올바른 값을 확인하려면 환경 변수 섹션을 확인하세요. 아래와 같이 **docker-compose.yml**을 수정하세요:
```
Comment (DaeHyeoNi) [inline docs/kr/README.md:73]:
```suggestion
필요에 따라 수정하고, 올바른 값을 확인하려면 환경 변수 섹션을 확인하세요. 아래와 같이 [docker-compose.yml](/docker-compose.yml)을 수정하세요:
```

**Review by DaeHyeoNi — CHANGES_REQUESTED** [review]

**Comment by DaeHyeoNi** [comment]
It's better to include only one change in a single PR. If possible

I recommend submitting separate PRs for the translation and the modification of the Docker command.

**Comment by thijsvanloef** [comment]
@DaeHyeoNi I agree on the splitting of PR's, however it's not really necessary for this one, since the change is so small

**Comment by lavi02** [comment]
Do I need to change something else? In the case of the last review, if I apply it, I think the actual content of the original document and the translated document seems to be slightly different.

**Review by DaeHyeoNi — APPROVED** [review]

**Pull request merged**